src/commands/deployments/editScmType.ts

```diff
@@ -11,7 +11,7 @@ import { ext } from "../../extensionVariables";
 import { ResolvedWebAppResource } from "../../tree/ResolvedWebAppResource";
 import { SiteTreeItem } from "../../tree/SiteTreeItem";
 
-export async function editScmType(context: IActionContext, node?: SiteTreeItem | DeploymentsTreeItem, newScmType?: ScmType, showToast?: boolean): Promise<void> {
+export async function editScmType(context: IActionContext, node?: SiteTreeItem | DeploymentsTreeItem, _nodes?: (SiteTreeItem | DeploymentsTreeItem)[], newScmType?: ScmType, showToast?: boolean): Promise<void> {
```

Comment:
I'm going to try to make it so we don't have refactor commands in this way. See https://github.com/microsoft/vscode-azuretools/pull/1246.

We can leave it for now, but once we make progress on https://github.com/microsoft/vscode-azuretools/pull/1246, we can undo these changes.